frontend/components/organisms/QuestionDetail/index.tsx

```diff
@@ -80,6 +84,19 @@ const QuestionDetail = ({
                                 {views_count} {views_count > 1 ? 'times' : 'time'}
                             </span>
                         </div>
+
+                        {team_name !== '' ? (
```

Comment:
I think can be improved. Just check if true, no need to use a ternary operation.

```suggestion
                        {team_name !== ' ' && (
```